test/gui/tst_sharing/test.feature

```diff
@@ -65,100 +31,34 @@ Feature: Sharing
         And user "Brian" has accepted the share "simple-folder" on the server offered by user "Alice"
         And user "Brian" has shared folder "Shares/simple-folder" on the server with user "Carol"
         And user "Brian" has set up a client with these settings and password "AaBb2Cc3Dd4":
-            """
-            [Accounts]
-            0\Folders\1\ignoreHiddenFiles=true
-            0\Folders\1\localPath=%client_sync_path%
-            0\Folders\1\paused=false
-            0\Folders\1\targetPath=/
-            0\Folders\1\version=2
-            0\Folders\1\virtualFilesMode=off
-            0\dav_user=brian
-            0\display-name=Brian
-            0\http_oauth=false
-            0\http_user=brian
-            0\url=%local_server%
-            0\user=Brian
-            0\version=1
-            version=2
-            [ownCloud]
-            remotePollInterval=5000
-            """
+        """
+        [ownCloud]
+        remotePollInterval=5000
+        """
```

Comment:
I think we can make another step  that explicitly states what poll interval to use.